### ISSUE TITLE
Allow custom overscan cropping

### ISSUE THREAD

**Issue by MuntKaicho** [xemu-project/xemu#1708]
### Feature Request

There are some games where there is a thin border on the outside of the game rendering area. Don't know if that is present on original hardware as well and gets cropped automatically or if it's an emulator rendering issue, but having an option to crop the output image based on user preferences, like PCSX2 does, would be a great way to get rid of it.

![image](https://github.com/user-attachments/assets/39dfe060-30ec-4f5b-a04f-5393af9f21ce)
![image](https://github.com/user-attachments/assets/5a0284f7-b268-4134-886a-ab5a915b5b5f)

**Comment by mborgerson** [comment]
I suspect the border in this case is actually an xemu bug, but I do agree overscan cropping would be a nice feature.

**Comment by antoniodesousa** [comment]
Yes, I also vouch for this feature to be added. Some games like Soulcalibur II has huge black borders on the top and bottom of the screen, which makes the image seems like squeezed. Having the option to crop those borders would be awesome. 

![xemu___v0 7 127_24_06_2024_16_29_05](https://github.com/user-attachments/assets/b7d071dd-96cc-4380-8df9-2653de25cd59)

**Comment by SlyCooperReloadCoded** [comment]
I opened #1262 a while ago which is essentially this, but just a toggle like PCSX2 has.  An Overscan toggle plus a pixels crop would be a nice combo.

**Comment by abaire** [comment]
I think this is indeed a dup of #1262
